### ISSUE TITLE
False positive: 'region' rule fails for <form> elements with non-empty "title" attribute

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#962]
This issue is related to #723 

```<form>``` elements with non empty title should be considered as ```form``` landmarks

- Demo:
```
<form id="1" title="name 3">A form element with a non-empty title</form>
```
<pre><code>
axe-core version: 3.0.3
axe-Coconut version: 3.4.0-alpha.2

</code></pre>

**Comment by WilcoFiers** [comment]
Yep, good find.

**Comment by marcysutton** [comment]
Here's what I found in AT testing ([demo here](https://codepen.io/marcysutton/pen/KBwxwv)):

- Safari and VO: forms aren't marked as landmarks at all, regardless of accessible name or implicit/explicit role. Only `<main>` is surfaced from the demo page.
- iOS Safari and VO: Using the landmarks rotor, I only hear "one landmark" with no clue about which one it is.
- Chrome and Voiceover: same as Safari.
- JAWS 2018 and IE11: Only `<div role="form"></div>` is a landmark, since the role is bolted on.
- NVDA and Firefox: works as expected:
   - `<div role="form">` is a landmark even without an accessible name.
   - `<form title="text">` is a landmark.
   - `<form>` is not a landmark.
   - `<form aria-label="text">` is a landmark.

That said, I think we can safely recommend code that matches the HTML and ARIA specs since this is a best practice. The results above are mainly for information purposes.
